components/merkledb/examples/migration/mod.rs

```diff
@@ -87,39 +76,50 @@ mod v1 {
     }
 }
 
-fn create_initial_data(fork: &Fork) {
-    const NAMES: &[&str] = &["Alice", "Bob", "Carol", "Dave", "Eve"];
-
-    let mut schema = v1::Schema::new(Prefixed::new("test", fork));
-    schema.ticker.set("XNM".to_owned());
-    schema.divisibility.set(8);
-
-    let mut rng = thread_rng();
-    for _ in 0..USER_COUNT {
-        let mut bytes = [0_u8; PUBLIC_KEY_LENGTH];
-        rng.fill(&mut bytes[..]);
-        let public_key = PublicKey::new(bytes);
-        let username = NAMES.choose(&mut rng).unwrap().to_string();
-        let wallet = v1::Wallet {
-            public_key,
-            username,
-            balance: rng.gen_range(0, 1_000),
-        };
-        schema.wallets.put(&public_key, wallet);
-
-        let history_len = rng.gen_range(0, 10);
-        schema
-            .histories
-            .get(&public_key)
-            .extend((0..history_len).map(|_| {
-                let mut bytes = [0_u8; HASH_SIZE];
-                rng.fill(&mut bytes[..]);
-                Hash::new(bytes)
-            }));
+/// Creates initial DB with some random data.
+pub fn create_initial_data() -> Arc<dyn Database> {
```

Comment:
Nit: Can't this method return `TemporaryDB`? I'd suggest to wrap it into an `Arc` on the caller side.